src/backend/jag-efiling-api/src/test/java/ca/bc/gov/open/jagefilingapi/document/DocumentApiImplTest.java

```diff
@@ -115,14 +113,17 @@ public void withValidIdReturnPayload() {
         assertEquals(actual.getBody().getNavigation().getCancel().getUrl(), CANCEL);
         assertEquals(actual.getBody().getNavigation().getError().getUrl(), ERROR);
     }
+
     @Test
     @DisplayName("CASE2: with null redis storage response return NotFound")
     public void withNullRedisStorageResponseReturnNotFound() {
 
-        when(redisStorageService.getByKey(any())).thenReturn(null);
+        when(redisStorageService.getByKey(any(), Mockito.any()))
```

Comment:
Not sure what it hates about this?

Reply:
any()?

Reply:
means that for any param passed in it will return null